src/MICore/LaunchOptions.cs

```diff
@@ -1792,6 +1792,27 @@ protected void InitializeCommonOptions(Json.LaunchOptions.BaseOptions options)
             this.WaitDynamicLibLoad = false;
 
             this.SourceMap = SourceMapEntry.CreateCollection(options.SourceFileMap);
+
+            if (options.SymbolLoadInfo != null)
+            {
+                this.WaitDynamicLibLoad = !String.IsNullOrWhiteSpace(options.SymbolLoadInfo.ExceptionList) || options.SymbolLoadInfo.LoadAll.HasValue;
```

Comment:
>options.SymbolLoadInfo.LoadAll.HasValue [](start = 110, length = 39)

This part doesn't seem to make sense: Why should we wait if LoadAll is set to `true`, which is the same as the default.

Reply:
So if we have an exception list or we are not loading all modules, we should wait on dynamic library load.
```suggestion
                this.WaitDynamicLibLoad = !String.IsNullOrWhiteSpace(options.SymbolLoadInfo.ExceptionList) || !options.SymbolLoadInfo.LoadAll.GetValueOrDefault(true);
```

Reply:
I *think* it only needs to be set if there is an exception list. If you don't have an exception list and load all is disabled -- we will load no symbols, but it doesn't look like we should need to somehow intervene.